webapp/components/searchable_user_list.jsx

```diff
@@ -18,9 +21,15 @@ export default class SearchableUserList extends React.Component {
         this.nextPage = this.nextPage.bind(this);
         this.previousPage = this.previousPage.bind(this);
         this.doSearch = this.doSearch.bind(this);
+        this.onSearchBoxKeyPress = this.onSearchBoxKeyPress.bind(this);
+        this.onSearchBoxChange = this.onSearchBoxChange.bind(this);
+
+        this.nextTimeoutId = 0;
 
         this.state = {
-            page: 0
+            page: 0,
```

Comment:
If we always have the page in the parent component, couldn't it just be passed in as a prop so we only need to track it in one place?